src/app/Profile.css

```diff
@@ -1,27 +1,38 @@
+/* Full-page centering and background color */
```

Comment:
This entire css file shouldnt be here, we are using tailwind css. If you can change it its not that deep